components/molecule/autosuggest/src/list/atom/item-featured.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-MoleculeAutosuggest-List-AtomItem'
+
+export const AtomListFeaturedItem = function (props) {
+  const {option, onClickCallback, onFocusCallback} = props
+  const cxNames = (option.isFocus())
+    ? cx(BASE_CLASS + '-row', BASE_CLASS + '-row-featured', BASE_CLASS + '-row-focus')
+    : cx(BASE_CLASS + '-row', BASE_CLASS + '-row-featured')
+
+  return (
+    <div
+      className={cxNames}
+      key={option.getPosition() + '-featured'}
+      onClick={onClickCallback(option)}
```

Comment:
You are calling the callback instead of passing it ?

Reply:
this callback is a function like `(option) => () => {}` so I can have the option on the new context